.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.11]
```

Comment:
Total smuggle but we can't really test against 3.8 if we spec >=3.11, so we have to choose one. I'd like to roll with 3.11...